api/src/opentrons/protocol_api/_types.py

```diff
@@ -8,3 +8,11 @@ class OffDeckType(enum.Enum):
 
 
 OFF_DECK: Final = OffDeckType.OFF_DECK
+
+# Set __doc__ manually as a workaround. When this docstring is written the normal way, right after
+# the constant definition, Sphinx has trouble picking it up.
+OFF_DECK.__doc__ = """\
+A special placeholder value for a labware's location, indicating that it's not currently on the robot's deck.
```

Comment:
```suggestion
A special location value, indicating that the labware is not currently on the robot's deck.
```
"placeholder" has a connotation of being _default_, which this definitely isn't.

Reply:
Good note, thanks.. Just changing "the labware" to "a labware."